packages/spruce-skill/.vscode/launch.json

```diff
@@ -43,7 +43,7 @@
 				"ORM_LOGGING": "true",
 				"API_ONLY": "true",
 				"TESTING": "true",
-				"DISABLE_MIGRATIONS": "true"
+				"DB_MIGRATIONS": "false"
```

Comment:
@kengoldfarb did we need to change this? going to impact @ryanrd and the ulta team down stream I'd imagine.

Reply:
`DISABLE_MIGRATIONS` has never been a real env var we're using. It's always been `DB_MIGRATIONS`

Reply:
sweet